### PR TITLE
adding instructions to readme on publishing videos to the website

### PR DESCRIPTION
**Pull request by ghost**

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -85,3 +85,10 @@ If you want to push code without deploying a new version of the site, make sure
   * See [Jenkins example post](/site/blob/master/_posts/2014-10-03-guide-to-deploying-artifacts-with-jenkins.md), especially the ``abstract`` field.
   * Note: comments can only be single line
   * Note: Even if they are too long, they'll be cut to 30 words
+
+### Adding a video to the publications
+
+  * Add an md file to the folder: `videos/_posts`
```
Comment (jukecraft) [inline README.md]: maybe add a comment that an existing entry can be used as a template

File: README.md [hunk 2]
```diff
@@ -85,3 +85,10 @@ If you want to push code without deploying a new version of the site, make sure
...
+  * Do not forget the attribute `video-url`, for youtube videos, use following format: https://www.youtube.com/embed/<video-id>
+  * Add an image to the video, for youtube videos, make a screenshot [here](http://youtubescreenshot.com/), the image is used on the home page and the carousel
```
Comment (jukecraft) [inline README.md]: where do we put the image?

File: README.md [hunk 3]
```diff
@@ -85,3 +85,10 @@ If you want to push code without deploying a new version of the site, make sure
...
+
```
Comment (jukecraft) [inline README.md:94]: what about the description? Do we take it straight from the YouTube listing?

**Review by jukecraft — COMMENTED** [review]
questions!

File: README.md (hunk 1 above)
Comment (ghost) [reply README.md]: I thought it was too obvious :)

File: README.md (hunk 3 above)
Comment (ghost) [reply README.md:94]: Well it depends on whatever the publisher feels like doing.. It will be reviewed anyway when PR is submitted I would say.

Comment (jukecraft) [reply README.md:94]: I meant that there is no guidance that you need one or how to come up with one. But I will merge this anyway. Just something to think about as additional info

**Pull request merged**